src/LogRecord.cpp

```diff
@@ -23,22 +23,12 @@
 #include "LogItInstance.h"
 #include "LogIt.h"
 
-/**
- * Rather irritating - boost needed for generating time-stamp: GCC 4.8 has
- * omitted the C++11 std::get_time/put_time that would obviate this necessity.
- *
- * GCC 5+ does though.
- */
-#include <boost/date_time/posix_time/posix_time.hpp>
-#include <boost/date_time/posix_time/posix_time_io.hpp>
-#include <boost/format.hpp>
-
-using boost::posix_time::time_facet;
-using boost::posix_time::microsec_clock;
+#include <chrono>
+#include <iomanip>
 
 using std::string;
 
-const string g_sTimestampFormat = "%Y-%m-%d %H:%M:%s";
+const string g_sTimestampFormat = "%Y-%m-%d %H:%M:%S";
```

Comment:
what is the effect of the change s -> S ?

Reply:
Hi Ben,

The variable is the same, but the implementation is a bit different.

Before (Boost) we were using `imboe` to specify the format we wish for our stream, and "%s" means (extracted from documentation) `Seconds with fractional seconds`. After that we give the stream the local time in microseconds.

In the STD implementation, I did not find a way to do it so concise as boost, so we are printing the local time, up to seconds (the reason behind %s -> %S) and after that appending us (microseconds) padding with zeros up to 6 digits.